app/views/content/train-to-be-a-teacher/how-to-apply-for-teacher-training.md

```diff
@@ -88,51 +88,32 @@ You can choose up to 4 teacher training courses for each application.
 
 If you do not find what you were looking for, there are [other ways to train](/train-to-be-a-teacher).
 
-Before you start, it’s particularly important to think about your [teacher training personal statement](/train-to-be-a-teacher/how-to-apply-for-teacher-training#writing-your-teacher-training-personal-statement).
+Before you start, it’s particularly important to think about your [teacher training personal statement](/train-to-be-a-teacher/how-to-apply-for-teacher-training#writing-your-personal-statement).
 
 Some people do not get on the course they want because their application has
```

Comment:
should we pop this into the same para as the above sentence maybe, to be clear that it's the personal statement we're talking about here?